### DISCUSSION TITLE
Share DNS Lists

### DISCUSSION THREAD

**Discussion by deathline94** [msasanmh/DNSveil#190] [General]
Hey y'all ...
i wanna share some dns lists that has over 250+ public doh servers 
it could be usable both as "custom servers" and the author could use these to update the built-in servers.
anyway , take a look and if you have other sources , share with the class :D

https://github.com/NiREvil/vless/blob/main/DNS%20over%20HTTPS/any%20DNS-over-HTTPS%20server%20you%20want.md#publicly-available-servers

and

https://github.com/curl/curl/wiki/DNS-over-HTTPS#doh---dns-over-https

**Comment by msasanmh** [comment]
Awesome, but be aware of servers with filters, like Google and Bing safe search, restricted youtube and blocked adult content.
Tools  ->  DNS Scanner can detect these filters. 

Thanks for sharing.

**Comment by msasanmh** [comment]
Maintained by Frank Denis:

https://github.com/DNSCrypt/dnscrypt-resolvers/blob/master/v3%2Fpublic-resolvers.md

**Comment by msasanmh** [comment]
Maintained by Adguard:

https://adguard-dns.io/kb/general/dns-providers/
